Can't issue SPDX (except spreadsheet)

**Describe the bug**
If the download location is http://www.google.com, OSS Notice cannot be issued in SPDX format except for spreadsheet.

```
Caused by: org.spdx.library.InvalidSPDXAnalysisException: Invalid download location pattern http://www.google.com. Must match the pattern ^(NONE|NOASSERTION|(((git|hg|svn|bzr)\+)?(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/|ssh:\/\/|git:\/\/|svn:\/\/|sftp:\/\/|ftp:\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+){0,100}\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*))|(git\+git@[a-zA-Z0-9\.]+:[a-zA-Z0-9/\\.@]+)|(bzr\+lp:[a-zA-Z0-9\.]+))$
web_1  | 	at org.spdx.library.model.SpdxPackage.setDownloadLocation(SpdxPackage.java:239)
web_1  | 	at org.spdx.library.model.SpdxPackage.<init>(SpdxPackage.java:111)
web_1  | 	at org.spdx.library.model.SpdxPackage$SpdxPackageBuilder.build(SpdxPackage.java:1036)
web_1  | 	at org.spdx.spreadsheetstore.PackageInfoSheetV2d2.getPackage(PackageInfoSheetV2d2.java:550)
web_1  | 	... 81 common frames omitted
```

--- a/build.gradle
+++ b/build.gradle
@@ -111,7 +111,7 @@ dependencies {
 	
 	implementation group: 'io.jsonwebtoken', name: 'jjwt-api', version: '0.11.2'
 
-	implementation group: 'org.spdx', name: 'tools-java', version: '1.0.2'
+	implementation group: 'org.spdx', name: 'tools-java', version: '1.0.3'
 
 	runtime group: 'io.jsonwebtoken', name: 'jjwt-impl', version: '0.10.7'
 	runtime group: 'io.jsonwebtoken', name: 'jjwt-jackson', version: '0.10.7'

